### DISCUSSION TITLE
Keyball61 OLED付近のジャンパ部分でパッドが剥離した場合の対処法について

### DISCUSSION THREAD

**Discussion by takashi0609** [Yowkees/keyball#504] [Q&A]
お世話になっております。
Keyball61(rev2)トラックボールなし側の基盤（左手）で、OLED付近のジャンパ部分について、4つ中、左からの3つで、パットが剥離してしまいました（添付画像の①-③）。

![IMG_1700](https://github.com/Yowkees/keyball/assets/42166968/720040de-b3bf-4572-b914-5f4fc67762b2)

対応方針については、調べた限り
・一番左（画像の①）は、#127及び#277より、オレンジの2つ
・左から3番目（画像の③）は、#266より、赤部分または、赤色の丸と黄緑色の丸
を繋げば良いかなと考えているのですが、以下質問させてください。

質問１）1番左と左から3番目については、上記の方針でよろしいでしょうか。
質問２）左から2番目（画像の②）については、どこを繋ぐべきでしょうか。

お手数をおかけ致しますが、アドバイス頂けますと幸いです。宜しくお願い致します。

**Comment by Ryo624** [comment]
ご連絡ありがとうございます。
OLEDのジャンパ部分のパッド剥離について了解しました。

質問１）1番左と左から3番目については、上記の方針でよろしいでしょうか。
①と③のジャンパについては上記の通りで繋いでいただければと思います。

質問２）左から2番目（画像の②）については、どこを繋ぐべきでしょうか。
②のジャンパについては添付しました写真の青色の部分を繋いでみてください。
![314406993-720040de-b3bf-4572-b914-5f4fc67762b2](https://github.com/Yowkees/keyball/assets/143698349/9971b9d8-de9d-44a4-a3e2-82256d59e0ae)

こちらの作業はタクトスイッチ、４Pソケット、TRRSコネクタのそれぞれをはんだ付けした後に行ってください。

**Reply by takashi0609** [reply]
ありがとうございます！無事に動作いたしました！

**Reply by Ryo624** [reply]
無事に解決していただきましてありがとうございます！
ぜひ末永くご愛用いただければ幸いです！
